extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -171,10 +172,12 @@ export class PublishDatabaseDialog {
 								title: constants.selectConnectionRadioButtonsTitle,
 								component: selectConnectionRadioButtons
 							},*/
+							/* TODO : enable using this when data source creation is enabled
```

Comment:
copy pasted comment?

Reply:
The comment updated but didn't added to the staged changes